templates/files/magento2/phpcs.xml

```diff
@@ -6,7 +6,5 @@
          xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">
     <description>PHPCS</description>
     <!--<exclude-pattern>path/to/exclude/*</exclude-pattern>-->
-    <rule ref="./vendor/youwe/coding-standard-magento2/src/Magento2"/>
-    <!-- Append arg -s to phpcs to display the name of failed sniffs -->
```

Comment:
Please do NOT remove this -s flag, it is really helpful in case you'd want to ignore a ruke
(must admit I'm not sure if <arg value="s"/> is now in some way part of <rule ref="YouweMagento2"/>)

Reply:
It's still in their but i moved it to the global youwe coding standards.